<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>iniciosesion</class>
 <widget class="QMainWindow" name="iniciosesion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titulo_app">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>1081</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Decotura ICG Inline</family>
      <pointsize>75</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Decotura ICG Inline;
font-size: 75pt;</string>
    </property>
    <property name="text">
     <string>Medical Solution</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="titulo_inicio">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>1081</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Decotura ICG</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Decotura ICG;
font-size: 30pt;</string>
    </property>
    <property name="text">
     <string>Inicio de sesión</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="introducir_user">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>310</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Decotura ICG</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Decotura ICG;
font-size: 30pt;
border-radius: 20px;
border: 3px solid #c0c0c0;
padding: 5px;</string>
    </property>
    <property name="placeholderText">
     <string>Usuario</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="introducir_pass">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>430</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Decotura ICG</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Decotura ICG;
font-size: 30pt;
border-radius: 20px;
border: 3px solid #c0c0c0;
padding: 5px;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Contraseña</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_is">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>550</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Decotura ICG</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Decotura ICG;
font-size: 30pt;
background-color: #0ff000;
border-radius: 20px;
color: white;</string>
    </property>
    <property name="text">
     <string>Iniciar sesión</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
